24-01-22/agenda.md

```diff
@@ -13,17 +13,24 @@ Any project updates from the MDN team
 ## Contributor updates
 
 Any project/work updates from contributors
-- [Community help needed] Documenting interoperable APIs. Let's document all `HTML*Element.*` APIs! [Instructions](https://github.com/orgs/mdn/discussions/476) – [Task list](https://docs.google.com/spreadsheets/d/1lvITVZJM8dx8s_Kee5UNI1hDfPNF0gXQwjPop5xIF5E/edit#gid=369545647)
+
+- [Community help needed] Documenting interoperable APIs. Let's document all `HTML*Element.*` APIs! (@teoli2003)
+  - [Instructions](https://github.com/orgs/mdn/discussions/476) & [Task list](https://docs.google.com/spreadsheets/d/1lvITVZJM8dx8s_Kee5UNI1hDfPNF0gXQwjPop5xIF5E/edit#gid=369545647)
 - [Community help needed] Document [missing JavaScript errors](https://github.com/mdn/mdn/issues/505) (@nataliaschlebinger)
 
 ## Questions/Discussions
 
 Any questions from folks & anything to discuss
 
-- [Using GFM-standard syntax for notes & co](https://github.com/mdn/yari/pull/10168) (like `[!Notes]`) 
-- Consensus secure context usage: [Where and how to use `{{securecontext_inline}}`](https://github.com/mdn/content/discussions/30546#discussioncomment-7796301)
-- The first APIs that need an "enrollment" before their use landed this week ([Shared Storage API](https://developer.mozilla.org/en-US/docs/Web/API/Shared_Storage_API)) How to display this information on the relevant pages? A banner for secure contexts?
-- Standards position in docs https://github.com/orgs/mdn/discussions/463
-- Firefox [experimental features](https://developer.mozilla.org/en-US/docs/Mozilla/Firefox/Experimental_features) page workflow
-- [Accepting more sources for polyfills](https://github.com/orgs/mdn/discussions/475#discussion-6000833)
-- How to get reviews on Yari PRs? (Many sit idle for months)
+**Carried over:**
+
+- Standards position in docs <https://github.com/orgs/mdn/discussions/463> (@Rumyra)
+- Consensus secure context usage: [Where and how to use `{{securecontext_inline}}`](https://github.com/orgs/mdn/discussions/482#discussioncomment-7825014) (@teoli2003)
+- The first APIs that need an "enrollment" before their use landed in ([Shared Storage API](https://developer.mozilla.org/en-US/docs/Web/API/Shared_Storage_API)) How to display this information on the relevant pages? A banner for secure contexts? (@teoli2003)
+
+**Added in January:**
+
+- [Using GFM-standard syntax for notes & co](https://github.com/mdn/yari/pull/10168) (like `[!Notes]`) (@teoli2003)
+- Firefox [experimental features](https://developer.mozilla.org/en-US/docs/Mozilla/Firefox/Experimental_features) page workflow (@pepelsbey)
```

Comment:
Can we move this to the top in the mozilla projects section? I realise this is out of scope, but saves a follow up pr

Reply:
>I realise this is out of scope, but saves a follow up pr 

All good, tnx. Will change now